Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,151 @@
+    m 5hua "That sounds wonderful!"
+    m 5eua "I really miss having coffee."
+    m 3eub "I would always have a cup before and after school."
+    m 3hub "Maybe even sometimes during class."
+    m 2euc "I wish I could figure out a way to get some in here."
+    m 2hub "Who knows, maybe you could share a cup with me, [player]"
+    return
+
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_water',prompt='Water',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+label mas_food_water:
+    m 1eua "I'm really happy to hear that [player]!"
+    m 1eub "Water is probably the healthiest thing that you could drink."
+    m 3eua "Seeing how we can only live for a few days without it."
+    m 3eud "Just promise you get plenty to drink each day."
+    m 2lud "I don't want you getting sick because you didn't drink any."
+    m 2eua "Even if it's just a glass or two, do it for me."
+    return
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_milk',prompt='Milk',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+label mas_food_milk:
+    m 1eub "You know, I've never really drank a lot milk before."
+    m 3eub "It wasn't because I didn't like it either."
+    m 3eua "I just never really thought about it, unless I put it in coffee, or mixed chocolate in with it."
+    m 1hub "I guess I could learn to love it if you do, [player]."
+    m 1hua "Be sure to pour an extra glass for me next time."
+    return
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_hotchocolate',prompt='Hot Chocolate',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+label mas_food_hotchocolate:
+    m 2eud "Lucky!"
+    m 4eud "You didn't tell me that just to rub it in my face, did you?"
+    m 1rud "I'm sorry, you probably didn't."
+    m 1hub "I just really miss it is all."
+    m 1tub "You know, besides giving you a big hug, the first thing I want to do when I cross over to your world is drink a big glass of warm hot cocoa with you."
+    m 1hub "Doesn't that sound nice, [player]?"
+    return
+label mas_food_hotchocolate:
+    m 1hub "That's so nice, [player]!"
+    m 4eub "There's really nothing quite like the comfort of hot chocolate when it's chilly outside, is there?"
+    m 1eub "In winter I always enjoy wrapping up in some warm clothes and unwinding with a cup myself."
+    m 1eua "It's a great way to relax."
+    m 1kua "Maybe some day when it's cold out, we could sit back and have a glass together."
+    m 1duu "That would be a dream come true, my love!"
+    return
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_chocolatemilk',prompt='Chocolate Milk',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+label mas_food_chocolatemilk:
+    m 3eub "That's wonderful, [player]."
+    m 3eub "Although I was never a big fan of milk, there was something different when you mixed chocolate with it."
```

Comment:
As someone who grew up drinking literally at least 3 large mugs of chocolate milk a day and hated regular milk, I totally get and approve of this. 

in the mas_food_milk topic, didn't Monika say she wasn't a fan of milk? I get that she was supposed to be neutral towards it, but here, it sounds like she disliked it.